tests/plugins/balloontoolbar/plugin.js

```diff
@@ -0,0 +1,26 @@
+/* bender-tags: balloontoolbar, 1346, context */
+/* bender-include: _helpers/default.js */
+/* global ignoreUnsupportedEnvironment */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
```

Comment:
`bender.editor = {};` creates default (based on bender tags) CKEditor instance, before any tests are executed. As you are already creating an editor instance in test (`bender.editorBot.create`) you may simply remove this line.